### DISCUSSION TITLE
Use Mac-Keys for Copy&Paste in Windows 11

### DISCUSSION THREAD

**Discussion by scharrin2** [utmapp/UTM#6299] [Ideas]
Hello,

It would be very helpful if the keys for Copy & Paste (Command+C / V / X) would also work in Windows 11 inside UTM. Now I always habe to switch between Command and Ctrl key.

This would be very helpful.

**Comment by deemoe404** [comment]
A workaround for now is that connect to the VM using Microsoft RDP client I think.

**Reply by scharrin2** [reply]
As you write „for now“ - is it planned to include the setting directly to UTM?

**Reply by deemoe404** [reply]
> As you write „for now“ - is it planned to include the setting directly to UTM?

Sorry for the confusion, but I am not the contributor of UTM. So I can't give you answer on this. 😢
